kartothek/core/utils.py

```diff
@@ -24,3 +24,10 @@ def _verify_metadata_version(metadata_version):
 
 def verify_metadata_version(*args, **kwargs):
     return _verify_metadata_version(*args, **kwargs)
+
+
+def _ensure_string_type(obj):
```

Comment:
this is a public function. Functions starting with `_` are module-private, e.g. should not be used outside of this module. Also, some docstrings might help.